floor/floor/controller/layout.py

```diff
@@ -1,15 +1,29 @@
 from builtins import object
 import json
+import os
 
 
 class Layout(object):
-    def __init__(self, config_filename):
+    DEFAULT_CONFIG_NAME = 'floor-layout'
```

Comment:
Moved this default into the Layout class, plus removed the need to include any path or ".json" components.